autoPyTorch/datasets/tabular_dataset.py

```diff
@@ -58,6 +82,15 @@ def __init__(self, X: Any, Y: Any,
                  train_transforms: Optional[torchvision.transforms.Compose] = None,
                  val_transforms: Optional[torchvision.transforms.Compose] = None,
                  ):
+
+        # TODO: Remove. Here for debug. If still here, it means
+        # that I forgot to clean this, but we need to discuss about
+        # standard scaling due to exploding gradient
```

Comment:
what is the issue here?

Reply:
Actually I didnt add this, @franchuterivera did. But it seems I should delete it. When we finished the API the performance was not so great so maybe he added it to test if this could be a reason.

Reply:
As the comment says, I forgot to remove this :), can you please remove this?